### ISSUE TITLE
[MCU] Build Process

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#34]
## User story
1. As a developer 
2. I want to have build process for compiling and deploying firmware using the Aurix IDE
3. So that we can efficiently manage and maintain the build process and its docmentation

## Acceptance criteria
* Provide a guide for building the MCU project using Aurix IDE
* Include troubleshooting tips and solutions for common build errors

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
